tests/e2e_regression/damask/e2e_config_1.yml

```diff
@@ -12,7 +12,7 @@ analysis:
     nodes:
       # Node will be accessed only when the filling up the cache.
       # In CI, the cache is expected to have all the node responses as the node is not reachable.
-      damask: { default: { rpc: unix:/tmp/node.sock } }
+      damask: { default: { rpc: internal-mainnet-archive-oasis-3.prd.oasis.io } }
```

Comment:
double checking, do we want to check this in?

Reply:
:+1: will remove. This ednpoint is not public (yet), firewalled for now.